### ISSUE TITLE
blacklist removal request from web (no space here) something india something

### ISSUE THREAD

**Issue by tripleee** [Charcoal-SE/SmokeDetector#1444]
Prolonged discussion here:

https://chat.stackexchange.com/transcript/11540?m=42080791#42080791

Tangentially, see also https://github.com/Charcoal-SE/metasmoke/issues/304

When the discussion got out of hand, I created a separate chat room for this topic, but it looks like the OP had already lost interest at that point.

https://chat.stackexchange.com/rooms/71171/blacklist-removal-discussion-metasmoke-304-smokedetector-1444

The current status is apparently that
* The OP is going to approach Github about getting the "offending" content removed;
* We will be waiting, probably in vain, for clarification on the OP's mandate to represent the site in question, and the precise scope of changes that would be acceptable to them

To do: summarize the pertinent points of the discussion and propose a workaround; but this is pending on input from their representative and the consensus of the Charcoal HQ community.

**Comment by ArtOfCode-** [comment]
Request denied unless further evidence comes to light.

https://chat.stackexchange.com/transcript/message/42084310#42084310

**Comment by tripleee** [comment]
One of the problems here was that they wanted us to remove their domain name from every discussion where they occurred. Some of the content which is already on Github is immutable and read-only (or at least completely out of _our_ control) and of course, even where we can remove mentions of their domain name, there has to be a really compelling reason to change the historical record (on the order of somebody's business secrets being leaked, I would think).

**Comment by AWegnerGitHub** [comment]
Further notes: 

weblineindia.com has mentioned as a spam/abuse site for several years. A quick search of "weblineindia.com spam" on Google turns up the following in the first five results:

- https://www.consumerforums.in/www-weblineindia-com-spam-fraud-stolen-clients-money - Complaints about stolen money - July 2016
- https://productforums.google.com/forum/#!topic/news/zCyoBuSzofQ - Report of spam like behavior to Google News - January 2012
- https://productforums.google.com/d/msg/news/lBb5E8X7cUU/N88Zred0LEEJ - Another report of spam like behavior to Google news - January 2011

---

Searching Google for just "weblineindia.com" does not return any of the above links in the first 5 Google search result pages.

**Comment by Undo1** [comment]
Changing the historical records they point to would cause a whole bunch of issues for us. We be rewriting (a lot of) Git history, which tends to make things unhappy for a long time. That's out of the question, I think. Which brings up another point - how much 'good' would removing it from the current file do if it's showing up in searches from commit records?

**Comment by tripleee** [comment]
Second-guessing what exactly they perceive to be problematic is probably quite pointless. In the chat discussion, I threw the OP a line asking if this can be demonstrated to harm their SEO metrics somehow, but s?he didn't bite. Now I'm more inclined to think they don't want users to google them and stumble into our tickets or source files, which probably look a lot more damning out of context than we ourselves might realize. What *we* actually mean by "blacklist" is a lot more benign and pedestrian than a casual visitor can be expected to understand.

**Comment by AWegnerGitHub** [comment]
I'm not second guessing. I'm providing context - outside of our own dataset in metasmoke - to demonstrate that this particular URL exhibits spam/abusive behavior in other locations around the internet *and* has done so for several years. 

The posts that are triggered by a search for this url in [metasmoke](https://metasmoke.erwaysoftware.com/search?utf8=%E2%9C%93&body_is_regex=1&body=www.weblineindia.com) returns 9 results in the past six months. Each of these links are posted by a 1 rep user. A few of those users were deleted after posting the link. 

This particular URL meets our [guidelines](https://charcoal-se.org/smokey/Guidance-for-Blacklisting-and-Watching#rules-for-blacklisting-and-watching) for blacklisting a website.

In short, this URL falls within our own standards blacklisting. 

I see no reason to argue semantics about the term "blacklisting". It does what users expect. It's a term/pattern/url that we've determined violates our guidelines and Stack Exchange's guidelines for spam. As such, we flag the suspected post and allow the community to deal with it. This particular URL has had low weights in all reports, with the *highest* reaching 168 - well below the threshold of automatic flagging.

**Comment by tripleee** [comment]
Completely agree. I was mainly responding to Undo -- that it's not clear if they *actually* want what they seem to be asking, because they sternly refused to clarify what exactly they perceived the problem to be. Ironically, I then contributed some speculation of my own; similarly to you, I'm just hoping to collect a useful understanding of whether their claim had any objective merit. At this point, I'm not really suggesting to even think about any concrete changes to how we do things, but I thought it was slightly thought-provoking to imagine how a SEO-sensitive site owner would react to finding their domain in a file which seems to be about bad behavior, blacklisting, penalties, and blocking.
